plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -474,6 +488,35 @@ public void downloadPlugins(List<Plugin> plugins) {
                 e.printStackTrace();
             }
         }
+
+        // Filter out failed plugins
+        final List<Plugin> failedPlugins = getFailedPlugins();
+        if (!skipFailedPlugins && failedPlugins.size() > 0) {
+            throw new DownloadPluginException("Some plugin downloads failed: " +
```

Comment:
is the temporary directory logged? / are these on disk so they can be inspected?

Reply:
They are on the disk, but the directory is not logged. Indeed it should